src/workloads/scale/CursorStormMongos.yml

```diff
@@ -0,0 +1,147 @@
+SchemaVersion: 2018-07-01
+Owner: Service Architecture
```

Comment:
@robsndr I just come across [this slack thread|https://mongodb.slack.com/archives/C0685CU3EPJ/p1708101747222789] suggesting the owner should be Query.

Reply:
Changed the owning team to Query.